VENDOR_NEUTRALITY.md

```diff
@@ -0,0 +1,33 @@
+# Metal3-io vendor neutrality declaration
+
+As a Cloud Native Computing Foundation project, Metal3-io will keep its
+[vendor neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/)
+
+**Hardware vendor neutrality:**
+As it is stated in the [documentation](https://book.metal3.io/) Metal3-io is a
+CNCF project with a goal to provide bare metal life cycle management
+functionality regardless of the vendor of the  hardware. The only requirement
```

Comment:
"The only requirements are" since we're specifying two requirements, BMC + protocol, unless you  want to change with BMC.... that supports either....

Reply:
done